webapp/tests/forms/steps/test_eligibility_steps.py

```diff
@@ -3274,3 +3272,138 @@ def test_if_no_user_b_elster_account_and_no_cheaper_check_then_set_no_info(self)
             step.handle()
 
         self.assertEqual(expected_information, step.render_info.additional_info['dependent_notes'])
+
+
+class TestEligibilityMaybeDisplaySteuerlotseStep(unittest.TestCase):
+    @pytest.fixture(autouse=True)
+    def attach_fixtures(self, test_request_context, app):
+        self.req = test_request_context
+        self.app = app
+
+    def test_if_user_b_has_no_elster_account_then_set_correct_info(self):
+        expected_information = [_('form.eligibility.result-note.user_b_elster_account-registration-maybe'),
+                                _('form.eligibility.result-note.deadline')]
+        session_data = {'marital_status_eligibility': 'married',
+                        'separated_since_last_year_eligibility': 'no',
+                        'user_a_has_elster_account_eligibility': 'yes',
+                        'user_b_has_elster_account_eligibility': 'no',
+                        'joint_taxes_eligibility': 'yes',
+                        'alimony_eligibility': 'no', }
+        with patch('app.forms.steps.eligibility_steps._', MagicMock(side_effect=lambda text_id: text_id)):
+            step = EligibilityMaybeDisplaySteuerlotseStep(
+                endpoint='eligibility',
+                stored_data=session_data,
+                render_info=EligibilityMaybeDisplaySteuerlotseStep.prepare_render_info(
+                    {})
+            )
+            step.handle()
+
+        self.assertEqual(expected_information, step.render_info.additional_info['dependent_notes'])
```

Comment:
This should actually never happen for the MaybeStep,  I think. If person B has no elster account, it will always be the success page that is shown, right?

Reply:
I guess the test we want here is `test_if_both_users_have_elster_account_then_set_correct_info`

Reply:
Thanks!